blockware/lib/VirtualBlox/platform_extra_script.py

```diff
@@ -0,0 +1,8 @@
+Import("env")
+
+# Override the "upload" target to run the local blockware
+env.AddCustomTarget(
+    "upload",
```

Comment:
I'm calling this the 'upload' target so that you can use the same 'upload' button in VSCode that you use to run the firmware on the real block.